modules/videopress/shortcode.php

```diff
@@ -22,6 +22,8 @@ protected function __construct() {
 
 		add_filter( 'oembed_fetch_url', array( $this, 'add_oembed_for_parameter' ) );
 
+		add_filter( 'the_content', array( $this, 'reverse_old_embeds' ) );
```

Comment:
Since we're filtering `the_content`, I think it would be nice to have unit tests to cover this.